database.json

```diff
@@ -228,11 +228,11 @@
     "repo": "cowsay",
     "desc": "Configurable talking cow for Deno"
   },
-  "crone": {
+  "cron": {
     "type": "github",
     "owner": "wrzonki",
-    "repo": "deno-crone",
-    "desc": "Simple crone scheduler for Deno"
+    "repo": "deno-cron",
+    "desc": "Simple cron scheduler for Deno"
```

Comment:
@ry [might not be happy](https://github.com/denoland/deno_website2/pull/486#discussion_r425933047) about this change 😄

Reply:
It was today at 2am - my bad.
Should I remove it, or wait for @ry to decide?

Reply:
I’d split it into two PRs